### ISSUE TITLE
Feature Request: Remove Truncation from Args when logging

### ISSUE THREAD

**Issue by marksalpeter** [jackc/pgx#998]
In `func logQueryArgs(args []interface{}` it looks like pgx deliberately truncates long args. We need to be able to use the full args to debug our queries in staging. 

Can we remove this truncation completely  or add a config param that would allow full args to be logger?

I'm happy to make the change in a pr. Just let me know which you prefer.

**Comment by jackc** [comment]
I don't think we should remove it entirely. That would flood the logs of people relying on that behavior now. Making the truncation length configurable would be reasonable though.
